### DISCUSSION TITLE
I want to fetch all accepted submitions by user on leetcode !!

### DISCUSSION THREAD

**Discussion by pandeyji711** [alfaarghya/alfa-leetcode-api#30] [Q&A]
Hello @alfaarghya,

I want to fetch all accepted submissions by a user on LeetCode. I have tried a lot, but it only gives the most recent 20 submissions. Please help!

**Comment by alfaarghya** [comment]
Hii @pandeyji711  👋 
LeetCode by default give us recent 20 submission, till now I am not able to find out how to fetch more than that,

**Comment by pandeyji711** [comment]
@alfaarghya 
is there any other alternative way??
